spec/raven/configuration_spec.rb

```diff
@@ -126,6 +127,25 @@
     end
   end
 
+  context 'being initialized without a release' do
+    before do
+      allow(File).to receive(:directory?).with(".git").and_return(false)
+      allow(File).to receive(:directory?).with("/etc/heroku").and_return(false)
+    end
+
+    it 'defaults to nil' do
+      expect(subject.release).to eq(nil)
+    end
+
+    it 'uses `SENTRY_CURRENT_ENV` env variable' do
```

Comment:
Test name doesn't match the actual variable name. Could please rename it so they are equal?

Reply:
Oops 🙈 
Fixed and force pushed